spaceship.zsh

```diff
@@ -738,6 +746,25 @@ spaceship_julia() {
     "$SPACESHIP_JULIA_SUFFIX"
 }
 
+# JAVA
+# Show current Java version
+spaceship_java() {
```

Comment:
Hmm, you're probably right. I do have multiple Java installations, but I **rarely** switch between them in this manner. When debugging through my IDE, I switch inside the IDE itself.

However, I do feel it's kind of neat to show it anyway, but if you feel it's redundant, you can close the issue and PR. One alternative is to default `SPACESHIP_JAVA_SHOW` to `false` (or not include it in `SPACESHIP_PROMPT_ORDER` by default).